### ISSUE TITLE
Fix menu dropdown effect

### ISSUE THREAD

**Issue by nylen** [ClassicPress/ClassicPress-Network#35]
With JavaScript disabled, or before the page has loaded the menu JavaScript, the menus look like this:

![2019-07-25T04 42 32-05 00](https://user-images.githubusercontent.com/227022/61864252-9c417480-aec0-11e9-9116-d853c4e7601d.png)

Not good - the menu dropdown effect should use CSS instead.

**Comment by EvoRay** [comment]
fixed line #288 display:none;

**Comment by nylen** [comment]
The specific issue I noted in the screenshot is fixed, but this still needs some work.  Now, the menus don't work at all without JavaScript enabled, and there is also some weird behavior around the hover effects (note that the submenu does not disappear unless the mouse moves into a specific area below the menu):

![cpnew-menu](https://user-images.githubusercontent.com/227022/62019354-0c484700-b1ae-11e9-9098-d0e05288ff9f.gif)

This isn't a huge issue in itself, but it tells me that when I look at the JS code used for the menu I'm probably not going to like it very much.

Basically there is no reason to use any JavaScript at all for this, a CSS `:hover` effect (and testing to make sure keyboard navigation is preserved) is all that is needed.

This same issue existed with the original child theme that you used as a starting point. I'll take this on if needed.

**Comment by EvoRay** [comment]
I'm not a competent JS dev, so please, be my guest. 
Here is a simple CSS-only menu that seems to work well:
https://codepen.io/philhoyt/pen/ujHzd

**Comment by EvoRay** [comment]
BTW, these latest issues identified also exist with the original CSS applied.

**Comment by nylen** [comment]
Another resource for this: https://css-tricks.com/solved-with-css-dropdown-menus/

The items that have submenus also need a visual indicator.

**Comment by timbocode** [comment]
I added a visual indicator for dropdown submenus in today's commit to remove "!important" rules.

**Comment by nylen** [comment]
This is fixed, thanks @Horlaes and @timbocode for working on it.

**Comment by timbocode** [comment]
Anyone else having issues with getting the dropdown to stay open?

![cpmenus](https://user-images.githubusercontent.com/4199514/64021785-a3e1e200-cb2c-11e9-8da6-f463919eab3f.gif)

**Comment by striebwj** [comment]
I am, I see it mostly on the community tab for some reason.

**Comment by johnalarcon** [comment]
Hovering the top-level items seems to work as expected. The trouble _seems_ to be that, when you move to hover a sub-item, if you aren't touching the sub-items actual text, the node loses focus. This is often related to links being used in a menu with their default display state (as links are inline elements.)

This behavior is often corrected by changing the display state for the anchors to `display:inline-block;` This causes the anchor to fill its container width and then the item can stay focused even if you move off the actual text of the individual item, so long as you don't move outside that particular anchor's bounding box.

Not tested here; just a super common scenario.

**Comment by nylen** [comment]
That was a nasty issue! Fixed now, thanks for reporting.
